calculate_average.js

```diff
@@ -0,0 +1,14 @@
+
+const calculateAverage = function calculateAverage(grades) {
```

Comment:
Could we DRY up this line a bit by making this an arrow function? `const calculateAverage = (grades) => {`